### DISCUSSION TITLE
How to enable dark mode

### DISCUSSION THREAD

**Discussion by aepanchi** [executablebooks/meta#1004] [Q&A]
Hi! 
I'd like to enable dark mode for my rST documentation. Where I can read about it and, more importantly, see examples of implementation? Including dependencies and changes to conf.py, css files, etc. 

Thanks in advance!

**Comment by lrq3000** [comment]
I would also appreciate a native dark mode style, as it is necessary for people with sleep/circadian rhythm disorders like me (and in general for anyone who prefer to reduce either the impact on their sleep or their eyestrain!).

Meanwhile, there are some documents viewers that can allow to read the MyST outputs in dark mode:

* Adobe PDF supports dark mode by changing some options.
* On Chrome, the Dark Reader module can convert into dark mode a lot of websites and documents, including PDFs read via Chrome's native PDF viewer.
* On mobile, Xodo and KOReader allow to read PDFs and ebooks in dark mode.
